tests/html/load-empty.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<!--
+    @license
+    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+  <head>
+    <title>load event Test for empty resources</title>
+    <script src="../../src/html-imports.js"></script>
+    <script>
+      WCT = {
+        waitFor: function(callback) {
+          HTMLImports.whenReady(callback);
+        }
+      };
+    </script>
+    <script src="../../../web-component-tester/browser.js"></script>
+    <script>
+      window.loadEvents = 0;
+      (function() {
+        function importLoaded(event) {
+          window.loadEvents++;
+          chai.assert.equal(event.type, 'load', 'event type should be `load`');
+          chai.assert.ok(event.target.import, 'link should have `import` defined');
+          // https://github.com/webcomponents/html-imports/issues/3
+          // chai.assert.ok(event.target.import instanceof Document, 'load event target can be used to find imported document');
```

Comment:
In the new polyfill we're experimenting the benefits of not having a separate Document created for each import, but rather just an Element. We can further discuss in the issue #3